README.md

```diff
@@ -1,4 +1,89 @@
+  <h3><b>Hello Microverse Project</b></h3>
 
+# 📗 Table of Contents
 
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
+# 📖 [Hello Microverse] <a name="about-project"></a>
 
+**[Hello Microverse]** is the first project in the first block of the first module of Microverse full stack web development program.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Please use the [Microverse README Template](https://github.com/microverseinc/readme-template) properly. Your previous code reviewer asked for the changes but as I can see you haven't implemented it yet. All the sub-section of the `getting-started` section are required and the *Microverse* README.md file is perfect for all kinds of projects and has everything to make your `README.md` file look professional.

- [ ] Kindly always look at `The README.md` file as a user manual that will help the developers or users of your repo to make them understand how they can use your project, and what they need before get start working on your project so other things like `Usage` and `Test` are important. Please look at the below image and consider adding the sub-sections.

![Screenshot from 2023-04-27 14-37-05](https://user-images.githubusercontent.com/66476329/234816379-3aa295ca-a423-40be-95d8-ee60253c615d.png)